test/loader.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env mocha */
```

Comment:
No, it is completely fine, to put it into one configuration file.
Adding comments does not scale if we will have multiple spec files.